src/System.Text.Encodings.Web/src/System/Text/Encodings/Web/TextEncoder.cs

```diff
@@ -19,6 +26,17 @@ public abstract class TextEncoder
         // yet consumption of concrete encoders is totally CLS compliant, 
         // as it’s mainly to be done by calling helper methods in TextEncoderExtensions class, 
         // and so I think the warning is a bit too aggressive.  
+        /// <summary>
```

Comment:
Nit: I'd suggest adding a blank line separating the normal comments from the XML comments